components/cluster/resource-agents/Makefile

```diff
@@ -13,24 +13,24 @@
 # Copyright 2019 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
 #
```

Comment:
Same here:
/export/home/andreas/oi-userland/components/cluster/resource-agents/build/manifest-i386-resource-agents.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/lib/heartbeat/send_arp \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.fullpath=lib/64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/gcc/7/lib/amd64/libssp.so.0'.
gmake: *** [/export/home/andreas/oi-userland/make-rules/ips.mk:518: /export/home/andreas/oi-userland/components/cluster/resource-agents/build/.resolved-i386] Error 1

Reply:
Same here. I have pushed the three fixes.

Reply:
Still many errors like this:
/export/home/andreas/oi-userland/components/cluster/pacemaker/build/manifest-i386-pacemaker.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/sbin/stonith_admin \
        pkg.debug.depend.type=elf \
        pkg.debug.depend.fullpath=lib/64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/gcc/7/lib/amd64/libssp.so.0 \
        pkg.debug.depend.fullpath=usr/openssl/1.1/lib/64/libssp.so.0'.
gmake: *** [/export/home/andreas/oi-userland/make-rules/ips.mk:518: /export/home/andreas/oi-userland/components/cluster/pacemaker/build/.resolved-i386] Error 1

Reply:
resource-agents publishes fine, though.

Reply:
libesmtp, too.

Reply:
I had copy-pasted the wrong line... GCC_* instead of GXX_* ...